### PR TITLE
fix: theme color injection workaround

### PR DESCRIPTION
**Pull request by kyrie25**
Workaround for the color injection problem that can be encountered when installing themes through Marketplace when no local theme is applied.
Also added the `preview` key for the Snippets part.

### CODE REVIEW

File: resources/install.sh [hunk 1]
```diff
@@ -34,6 +34,12 @@ cd "$INSTALL_DIR/marketplace"
 # Remove old custom app name if exists
 spicetify config custom_apps spicetify-marketplace-
 
+# Color injection fix
+spicetify config inject_css 1
+spicetify config replace_colors 1
+spicetify config current_theme SpicetifyDefault
+spicetify config color_scheme SpicetifyDefault
```
Comment (theRealPadster) [inline resources/install.sh:41]: Is this a thing? Mine worked with the colour scheme blank

File: resources/install.sh [hunk 2]
```diff
@@ -34,6 +34,12 @@ cd "$INSTALL_DIR/marketplace"
 # Remove old custom app name if exists
 spicetify config custom_apps spicetify-marketplace-
 
+# Color injection fix
+spicetify config inject_css 1
+spicetify config replace_colors 1
+spicetify config current_theme SpicetifyDefault
```
Comment (theRealPadster) [inline resources/install.sh:40]: Is there a way to check if they have the colour scheme set? People will get confused if they install marketplace and then lose their theme.

File: README.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 # spicetify-marketplace
 
-Download extensions and themes directly from within [Spicetify](https://github.com/spicetify/spicetify-cli). 
+Download extensions and themes directly from within [Spicetify](https://github.com/spicetify/spicetify-cli).
 
 Made by [@theRealPadster](https://github.com/theRealPadster) and [@CharlieS1103](https://github.com/CharlieS1103).
 
-Based on the [reddit Custom App](https://github.com/spicetify/spicetify-cli/wiki/Custom-Apps#reddit)  
+Based on the [reddit Custom App](https://github.com/spicetify/spicetify-cli/wiki/Custom-Apps#reddit)
```
Comment (theRealPadster) [inline README.md:7]: Can we remove this? I feel like it's diverged enough and there's not really much point adding this.

Comment (CharlieS1103) [reply README.md:7]: I agree with this @theRealPadster , plus lyrics-plus and new-releases both use the same system

File: README.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
 # spicetify-marketplace
 
-Download extensions and themes directly from within [Spicetify](https://github.com/spicetify/spicetify-cli). 
+Download extensions and themes directly from within [Spicetify](https://github.com/spicetify/spicetify-cli).
 
 Made by [@theRealPadster](https://github.com/theRealPadster) and [@CharlieS1103](https://github.com/CharlieS1103).
```
[the remark this answers is not in the document]
Comment (CharlieS1103) [reply README.md:5]: Agreed

File: resources/install.sh (hunk 2 above)
Comment (kyrie25) [reply resources/install.sh:40]: Seems like it is possible, will patch in a sec.

Comment (CharlieS1103) [reply resources/install.sh]: @kyrie25 I don't run windows unfortunately.

**Review by theRealPadster — APPROVED** [review]
I think this is good, right? The bash syntax checks out

**Comment by kyrie25** [comment]
Don't think there are anything else to do. If the bash script works then it should be ready to merge.

**Pull request merged**